litestar/contrib/sqlalchemy/repository/_async.py

```diff
@@ -906,7 +912,9 @@ def _apply_filters(
             The select with filters applied.
         """
         for filter_ in filters:
-            if isinstance(filter_, LimitOffset):
+            if isinstance(filter_, ColumnElement):
```

Comment:
We might want to refactor this to a map based dispatch at some point instead of this long list of `if` statements